src/app_main.c

```diff
@@ -154,22 +154,30 @@ bool extract_bip32(uint8_t *depth, uint32_t path[5], uint32_t rx, uint32_t offse
 bool validate_bnc_bip32(uint8_t depth, uint32_t path[5]) {  // path is 10 bytes for compatibility
     // Only paths in the form 44'/714'/{account}'/0/{index} are supported
     // placeholder for a user-specified value is 0xFFFFFFFF
-    uint32_t placeholder = 0xFFFFFFFF;
+    uint8_t mutable_nodes[] = {2, 4}; // account, index
```

Comment:
The functionality is ok. Just a small suggestion:

```
 uint8_t mutable_nodes[] = {2, 4}; // account, index
```
```
 bool is_mutable_nodes[] = {false, false, true, false, true}; // account at 2, index at 4
```
How about replacing `mutable_nodes` with `is_mutable_nodes`, and then we can simplify below code.
```
        mutable = false;
        for (j = 0; j < sizeof(mutable_nodes); j++) {
            if (i != mutable_nodes[j]) continue;
            mutable = true;
            break;
        }
        if (mutable) continue;
```
```
        if (is_mutable_nodes[i]) continue;
```

Reply:
Ok, sure